npm-shrinkwrap.json

```diff
@@ -2124,7 +2139,7 @@
         "xtend": {
           "version": "4.0.1",
           "from": "xtend@>=4.0.0 <5.0.0",
-          "resolved": "https://registry.npmjs.org/xtend/-/xtend-4.0.1.tgz",
+          "resolved": "https://lunabuild.akamai.com/nexus/content/groups/npm-all/xtend/-/xtend-4.0.1.tgz",
```

Comment:
Ug -- yeah was just reading through some of this... tried clearing npm cache, but nothing yet